systemd/units/unittypes.yaml

```diff
@@ -0,0 +1,13 @@
+# Valid systemd unit types according to SYSTEMD.UNIT(5)
+
+Valid:
+  - service
+  - socket
+  - device
+  - mount
+  - automount
```

Comment:
I think we missed `systemd.swap(5)` unit here.

Reply:
Good catch, fixed.